tests/helpers.py

```diff
@@ -949,6 +949,86 @@ def measure_pvc_creation_time(interface, pvc_name):
     return total.total_seconds()
 
 
+def measure_pvc_creation_time_bulk(interface, pvc_obj_list):
+    """
+    Measure PVC creation time of bulk PVC based on logs.
+
+    Args:
+        interface (str): The interface backed the PVC
+        pvc_obj_list (list): List of PVC Names for measuring creation time
```

Comment:
`pvc_obj_list` might be misleading as they are not objects and just names. can you change the name of this parameter?

Reply:
It's pvc_obj_list only, if we expect pvc_name_list then while passing need to get name of each pvc object and store in list. I will change the args description, in next commit.

Reply:
changed it back to pvc_name_list, resolving comment.